arcgis-ios-sdk-samples/Scenes/Scene symbols/SceneSymbolsViewController.swift

```diff
@@ -63,14 +63,14 @@ class SceneSymbolsViewController: UIViewController {
         let diamondSymbol = AGSSimpleMarkerSceneSymbol(style: .diamond, color: .random(), height: 200, width: 200, depth: 200, anchorPosition: .center)
         let sphereSymbol = AGSSimpleMarkerSceneSymbol(style: .sphere, color: .random(), height: 200, width: 200, depth: 200, anchorPosition: .center)
         let tetrahedronSymbol = AGSSimpleMarkerSceneSymbol(style: .tetrahedron, color: .random(), height: 200, width: 200, depth: 200, anchorPosition: .center)
-        
```

Comment:
I meant to comment on this earlier. Could you please revert the changes to this file? Putting the cursor on the line and pressing Ctrl+I (Editor > Structure > Re-Indent) should indent the lines properly. I think that's how they were before.